### PR TITLE
v1.3.1

### PR DESCRIPTION
**Pull request by byanko55**
**Bug Fixes**
* E-mail button open a 404 page

**Minor Update**
* Always display search/theme selector button on desktop window

### CODE REVIEW

**Comment by byanko55** [comment]
Refer this issue #40

File: _includes/sidebar.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h3>Contact</h3>
                     </a></li>
                 {% endif %}
                 {% if site.email %}
-                    <li><a aria-label="My Email" href="{{ site.email }}">
+                    <li><a aria-label="My Email" href="mailto:{{ site.email }}">
```
Comment (cvlian) [inline _includes/sidebar.html:35]: Same could happen in your [another theme template](https://github.com/byanko55/jekyll-professional-resume).

Comment (byanko55) [reply _includes/sidebar.html:35]: It's like you said.

File: _sass/sidebar.scss [hunk 2]
```diff
@@ -302,14 +302,18 @@
   }
 }
```
Comment (cvlian) [inline _sass/sidebar.scss:304]: I had not getting noticed those buttons were hiding to the right. Not sure what you were meant to, that would be "TOTAL DISRESPECT" for laptop user!

**Review by cvlian — APPROVED** [review]

**Comment by mgarrixx** [comment]
I already have done build tests. You can see them [here](https://mgarrixx.github.io/).

**Review by mgarrixx — APPROVED** [review]

**Pull request merged**